cartoframes/core/cartodataframe.py

```diff
@@ -175,5 +169,37 @@ def set_geometry_from_xy(self, x, y, drop=False, inplace=False, crs=None):
 
         return frame
 
-    def has_geometry(self):
-        return self._geometry_column_name in self
+    @property
```

Comment:
these method are here to avoid losing the CartoDataframe instance, right?

Reply:
Yes, these methods are wappers for the GDF and DF classes